sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/util/XrayWrapperConfigurationHelper.java

```diff
@@ -19,10 +19,10 @@ public static XrayWrapperConfiguration createXrayConfiguration(XrayWrapperScanTy
         String zipDirectory = environmentVariableReader.readEnvAsString(EnvironmentVariableConstants.WORKSPACE_ENV) + "/XrayArtifactoryReports";
 
         // get repository according to scan type
-        String repository = "";
+        String registry = "";
         if (scanType.equals(XrayWrapperScanTypes.DOCKER)) {
-            repository = environmentVariableReader.readEnvAsString(EnvironmentVariableConstants.DOCKER_REGISTRY_ENV);
+            registry = environmentVariableReader.readEnvAsString(EnvironmentVariableConstants.DOCKER_REGISTRY_ENV);
         }
-        return XrayWrapperConfiguration.Builder.create(artifactoryUrl, repository, zipDirectory, outputFile).build();
+        return XrayWrapperConfiguration.Builder.builder(artifactoryUrl, registry, zipDirectory, outputFile).build();
```

Comment:
is it correct that we accept registry as an empty string here?
shoud we throw instead an `IllegalStateException` if the scan type is not supported (something else than docker) ?

Or at least a log warning here?